File: CrystalCards.Models/Card.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace CrystalCards.Models
{
    public class Card
    {
        public Card()
        {
            Points=new List<NPPoint>();
            ActionPoints=new List<ActionPoint>();
            Links=new List<Link>();
        }

        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public IList<NPPoint> Points { get; set; }
        public IList<ActionPoint> ActionPoints { get; set; }
        public IList<Link> Links { get; set; }
        public int Order { get; set; }

        [ForeignKey("ProjectId")]
        public int? ProjectId { get; set; }
    }
}

File: CrystalCards-Web/src/app/card.ts
import {NPPoint} from './cards/NPPoint';
import {ActionPoint} from './cards/ActionPoint';
import {UrlLink} from './cards/Link';

export class Card {
    id:number;
    title: string;
    description: string;
    npPoints: NPPoint[];
    order: number;
  actionPoints: ActionPoint[];
  links:UrlLink[];
}

